dashboard/lib/tasks/seed.rake

```diff
@@ -148,9 +148,8 @@ namespace :seed do
     NO_PARTNER = 'NO PARTNER'
     expected_count = `grep -v '#{NO_PARTNER}' #{mapping_tsv} | wc -l`.to_i - 1
     raise "#{mapping_tsv} contains no data" unless expected_count > 0
-
     RegionalPartnersSchoolDistrict.transaction do
-      if RegionalPartnersSchoolDistrict.count < expected_count
+      if RegionalPartnersSchoolDistrict.count <= expected_count
```

Comment:
per offline discussion, this check is in here to skip this step (takes 1 minute) during rake seed in the common case. Because updating (rather than adding) is rare, we should leave this as `<` and do a manual step when we need to update a row without adding any rows.

One option for this is to add a `force_regional_partners_school_districts` task which (1) is not part of `rake seed:all`, (2) is run manually when this csv is updated without adding rows, and which looks the same as `regional_partners_school_districts` but without the `<` check protecting the call to `find_or_create_all_from_tsv`.